packages/gestalt/src/Checkbox.css

```diff
@@ -35,13 +37,13 @@
 }
 
 .checkSm {
-  border-radius: 3px;
+  border-radius: 4px;
   height: var(--size-sm);
   width: var(--size-sm);
 }
 
 .checkMd {
-  border-radius: 4px;
+  border-radius: 8px;
```

Comment:
@AlbertCarreras Instead, can we compose from `rounding2`?

```
composes: rounding2 from "./Borders.css";
```

Reply:
DONE